bolt/internal/internal.go

```diff
@@ -98,6 +98,14 @@ func MarshalLayout(l chronograf.Layout) ([]byte, error) {
 			}
 		}
 
+		axes := make(map[string]*Axis, len(c.Axes))
+		for a, r := range c.Axes {
```

Comment:
Yeah, originally these were `DashboardRange`s, and we later renamed them to `Axes` during a review. Given that both `Axes` and `Axis` start with `a`, and convention is to use single-letter vars for short loops, I left it as `r`.